test/k8sT/manifests/demo.yaml

```diff
@@ -46,6 +51,12 @@ spec:
           httpGet:
             path: /
             port: 80
+      - name: udp
+        image: docker.io/cilium/echoserver-udp:v2020.01.30
```

Comment:
We should cache the image (= update the list of Docker images in GH/cilium/packer-ci-build).

Reply:
https://github.com/cilium/packer-ci-build/pull/185